dlc-wallet/src/main/resources/postgresql/dlc/migration/V1__dlc_db_baseline.sql

```diff
@@ -1,108 +1,144 @@
-CREATE TABLE "wallet_dlcs"
+CREATE TABLE "global_dlc_data"
 (
-    "param_hash"       TEXT    NOT NULL UNIQUE,
-    "temp_contract_id" TEXT    NOT NULL UNIQUE,
-    "contract_id"      TEXT UNIQUE,
-    "state"            TEXT    NOT NULL,
-    "is_initiator"     BOOLEAN NOT NULL,
-    "account"          TEXT    NOT NULL,
-    "key_index"        INTEGER NOT NULL,
-    "oracle_sigs"      TEXT,
-    "funding_outpoint" TEXT,
-    "funding_tx_id"    TEXT,
-    "closing_tx_id"    TEXT,
-    "outcomes"         TEXT,
-    "oracles_used"     TEXT,
-    constraint "pk_dlc" primary key ("param_hash")
+    "dlc_id"              TEXT PRIMARY KEY,
+    "temp_contract_id"    TEXT    NOT NULL UNIQUE,
+    "contract_id"         TEXT UNIQUE,
+    "protocol_version"    INTEGER NOT NULL,
+    "state"               TEXT    NOT NULL,
+    "is_initiator"        INTEGER NOT NULL,
+    "account"             TEXT    NOT NULL,
+    "change_index"        INTEGER NOT NULL,
+    "key_index"           INTEGER NOT NULL,
+
+    "funding_outpoint"    TEXT,
+    "funding_tx_id"       TEXT,
+    "closing_tx_id"       TEXT,
+    "aggregate_signature" TEXT
 );
-CREATE
-INDEX "wallet_dlcs_param_hash_index" on "wallet_dlcs" ("param_hash");
 
-CREATE TABLE "oracle_announcements"
+CREATE TABLE "contract_data"
 (
-    "announcement" TEXT NOT NULL PRIMARY KEY,
-    "pub_key"      TEXT NOT NULL
+    "dlc_id"                TEXT PRIMARY KEY,
+    "oracle_threshold"      INTEGER NOT NULL,
+    "oracle_params"         TEXT,
+    "contract_descriptor"   TEXT    NOT NULL,
+    "contract_maturity"     INTEGER NOT NULL,
+    "contract_timeout"      INTEGER NOT NULL,
+    "total_collateral"      INTEGER NOT NULL,
+    "fee_rate"              TEXT    NOT NULL,
```

Comment:
nit: fee_rate seems like global data and not contract data imo (seeing as we could in the future have an rbf workflow like dual funded channels)